pkg/cloudprovider/vsphere/vsphere_test.go

```diff
@@ -216,6 +217,113 @@ func TestVSphereLoginByToken(t *testing.T) {
 	vcInstance.Conn.Logout(ctx)
 }
 
+func TestAlphaDualStackConfig(t *testing.T) {
+	var vs *VSphere
+	var (
+		username = "user"
+		password = "password"
+	)
+	var testcases = []struct {
+		testName                 string
+		conf                     string
+		enableDualStackFeature	 bool
+		expectedIsSecretProvided bool
+		expectedUsername         string
+		expectedPassword         string
+		expectedError            error
+		expectedThumbprints      map[string]string
+	}{
+		{
+			testName: "Verifying dual stack env var required when providing two ip families",
+			conf: `[Global]
+			user = user
+			password = password
+			datacenters = us-west
+			[VirtualCenter "0.0.0.0"]
+			user = user
+			password = password
+			ip-family = ipv6,ipv4
+			
+			`,
+			enableDualStackFeature: false,
+			expectedUsername: username,
+			expectedPassword: password,
+			expectedError:    vcfg.ErrMissingDualStackEnvVar,
+		},
+		{
+			testName: "Verifying dual stack env var existing when providing two ip families",
+			conf: `[Global]
+			user = user
+			password = password
+			datacenters = us-west
+			[VirtualCenter "0.0.0.0"]
+			user = user
+			password = password
+			ip-family = ipv6,ipv4
+			
+			`,
+			enableDualStackFeature: true,
+			expectedUsername: username,
+			expectedPassword: password,
+			expectedError:    nil,
+		},
+		{
+			testName: "Dual stack env var not required when providing single ip family",
+			conf: `[Global]
+			user = user
+			password = password
+			datacenters = us-west
+			[VirtualCenter "0.0.0.0"]
+			user = user
+			password = password
+			ip-family = ipv6
+			
+			`,
+			enableDualStackFeature: false,
+			expectedUsername: username,
+			expectedPassword: password,
+			expectedError:    nil,
+		},
+	}
+	for _, testcase := range testcases {
+		t.Logf("Executing Testcase: %s", testcase.testName)
```

Comment:
Use t.Run to wrap the test case instead of logging the name here, something like:
```
t.Run(testcase.testName, func(t *testing.T) {
    ....
})
```